server/controllers/events/events_controller_e2e_test.go

```diff
@@ -1365,9 +1365,9 @@ func mkSubDirs(t *testing.T) (string, string, string) {
 
 // Will fail test if conftest isn't in path and isn't version >= 0.25.0
 func ensureRunningConftest(t *testing.T) {
-	localPath, err := exec.LookPath(fmt.Sprintf("conftest%s", ConftestVersion))
+	localPath, err := exec.LookPath("conftest")
 	if err != nil {
-		t.Logf("conftest >= %s must be installed to run this test", ConftestVersion)
+		t.Logf("conftest must be installed to run this test")
```

Comment:
version is greater than or equal logic is already implemented. so just remove `conftest$version` command dependency using `conftest` command.

https://github.com/runatlantis/atlantis/blob/b6f372416fab3b06cc843857a8e5a26a98e7cf93/server/controllers/events/events_controller_e2e_test.go#L1366-L1392